zip-0209.rst

```diff
@@ -0,0 +1,48 @@
+::
+
+  ZIP: 209
+  Title: Prohibit Negative Shielded Value Pool
+  Author: Sean Bowe <sean@z.cash>
+  Category: Consensus
+  Created: 2019-02-25
+  License: MIT
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+The "Sprout value pool balance", as implied by section 4.11 of the Zcash Protocol Specification [#protocol]_, is the sum of all previous transactions' ``vpub_old`` fields subtracted by the sum of all previous transactions ``vpub_new`` fields.
+
+The "Sapling value pool balance", as implied by section 4.12 of Zcash Protocol Specification [#protocol]_, is the negation of the sum of all previous transactions' ``valueBalance`` fields.
+
+Abstract
+========
+
+This proposal defines how the consensus rules are altered such that blocks which produce negative shielded value pools are prohibited.
+
+Motivation
+==========
+
+It is possible for nodes to monitor the total amount of coins that are shielded or unshielded in each of the Sprout or Sapling shielded transaction protocols. If the amount of coins that are unshielded exceed the amount that are shielded, a balance violation has occurred in the corresponding shielded transaction protocol.
```

Comment:
I don't think it's correct to say the 'amount of coins'. We could say something like:

In both the Spout and Sapling shielded transaction protocols, nodes are able to monitor the values of coins which are shielded and unshielded. If the value coins which are unshielded exceeds the value of coins which are unshielded,  a balance violation has occurred in the corresponding shielded transaction protocol.

Reply:
I don't agree with this rewording. I think "amount of coins" is correct. "Coins" don't have value in our system, they just refer to some unit(s) of ZEC.

Reply:
```suggestion
It is possible for nodes to monitor the total value of notes that are shielded to, or unshielded from,
each of the Sprout or Sapling value pools. If the total value that is unshielded exceeds the total value
that was shielded for a given pool, a balance violation has occurred in the corresponding shielded
transaction protocol.
```